packages/uniforms/src/BaseForm.tsx

```diff
@@ -201,7 +201,7 @@ export default class BaseForm extends Component<any, any> {
     return changedKeys(root, valueA, valueB);
   }
 
-  getNativeFormProps() {
+  getNativeFormProps(): { [prop: string]: unknown } {
```

Comment:
There is a nice utility type provided by typescript: [`Record<K,T>`](https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkt). So you could use `Record<string, unknown>`.